### PR TITLE
FIX: add link to documentation and other doc fixes

### PR DESCRIPTION
**Pull request by awalter-bnl**
This adds the link to the suitcase documentation and afixes a few other doc-string issues

### CODE REVIEW

File: suitcase/json_metadata/__init__.py [hunk 1]
```diff
@@ -67,19 +69,29 @@ def export(gen, directory, file_prefix='{uid}-', cls=event_model.NumpyEncoder,
...
+
+    Generate files with unique-identifier names in the current directory.
+
     >>> export(gen, '')
+
     Generate files with more readable metadata in the file names.
+
     >>> export(gen, '', '{plan_name}-{motors}-')
+
     Include the experiment's start time formatted as YY-MM-DD_HH-MM.
```
Comment (mrakitin) [inline suitcase/json_metadata/__init__.py]: Ah, we still have `YY` here. Can be fixed later, anyway.

Comment (danielballan) [reply suitcase/json_metadata/__init__.py]: Thanks for checking. Just opened https://github.com/NSLS-II/suitcase-json-metadata/pull/5

Comment (mrakitin) [reply suitcase/json_metadata/__init__.py]: Thanks, it's a very annoying synchronization :)

Comment (danielballan) [reply suitcase/json_metadata/__init__.py]: Indeed. The downside of our "many small packages" approach, but even so I still feel strongly that the benefits outweigh the costs.

Comment (mrakitin) [reply suitcase/json_metadata/__init__.py]: Yeah, the flexibility and individual release cycles matter.

conda-forge has the same problem, when a new feedstock repo is configured from the "current" (at some point of time) state of the staging repo, which than has to be updated if the master repo is updated.

**Pull request merged**